Common/UnitDefinitions/SpecificFuelConsumption.json

```diff
@@ -3,6 +3,11 @@
   "BaseUnit": "GramPerKiloNewtonSecond",
```

Comment:
Out of scope of PR, but we should probably rename this to `Kilonewton` with lowercase `n`.

Reply:
Added to https://github.com/angularsen/UnitsNet/issues/1200